src/main/java/org/apache/mesos/specification/TaskSet.java

```diff
@@ -0,0 +1,11 @@
+package org.apache.mesos.specification;
+
+import java.util.List;
+
+/**
+ * A TaskSet groups a set of Tasks that should be deployed using the same deployment strategy.
```

Comment:
Maybe specify how in practice a TaskSet corresponds to a Phase(correct term?) in Plans, where each TaskSpecification in turn corresponds to a Block.